### PR TITLE
Uptest testing tool support

### PR DESCRIPTION
**Pull request by flipchan**
Uptest RFP

### CODE REVIEW

File: open_rfps/uptest.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+Total amount of hours(0+10+30+30+30): 100
+
+## Payment terms:
+USD price of each milestone to be converted into XOR at delivery. 
+Receivers XOR address: oFFniTXNGeA9b8W62e1gsvz6zHPAqH4MfdZ79JbbvWvNxsfNc
```
Comment (WRRicht3r) [inline open_rfps/uptest.md]: Hi, this doesn't appear to be a SORA network address (addresses start with cn)

Comment (flipchan) [reply open_rfps/uptest.md]: I checked on  polkadot.js apps and it gave me this address, will recheck

Comment (flipchan) [reply open_rfps/uptest.md]: @WRRicht3r checked and updated the address

**Comment by flipchan** [comment]
closing rfp @WRRicht3r , no updates in a bit and uptest is not the main substrate tool that i allocate my energy to

**Pull request closed without merging**